extensions/qute/deployment/src/test/java/io/quarkus/qute/deployment/records/TemplateRecordTest.java

```diff
@@ -69,6 +71,9 @@ public void testTemplateRecords() throws InterruptedException, ExecutionExceptio
 
         assertEquals("Hello Lu!", new helloWorld("Lu").render());
         assertEquals("Lu", new hello$name("Lu").render());
+
+        assertEquals("15:true:foo", new multiParams(true, 15, "foo").render());
+        // assertThrows(IllegalArgumentException.class, () -> new multiParams(false, 50, null));
```

Comment:
?

Reply:
Ah, I forgot to uncomment this assertion after I fixed the problem with compact record constructor...